hnn/simdata.py

```diff
@@ -231,7 +231,7 @@ class SimData(object):
 
     def __init__(self):
         self._sim_data = {}
-        self._opt_data = {}
+        self._opt_data = {'initial_dpl': None, 'initial_error': 1e9}
```

Comment:
Is this a placeholder or is there a reason for using `1e9`? If not I would just got with `float`